RELEASE.md

```diff
@@ -4,15 +4,15 @@ The following is the release process you should follow to publish a new version
 
 ## Update The Changelog
 
-First, we need to update the `CHANGELOG.md` file for the project. We do this via the [GitHub Changelog Generator](https://github.com/skywinder/github-changelog-generator/), using the following command:
+First, we need to update the `CHANGELOG.md` file for the project. We do this via the [lerna-changelog](https://github.com/lerna/lerna-changelog). This requires all PRs to be labeled appropriately. Use the following command to generate the changelog from the most recent tag:
 
 ```bash
-github_changelog_generator --future-release vx.x.x
+yarn changelog
 ```
 
-Where you replace the `x.x.x` with the appropriate version you are publishing.
+Copy the output into `CHANGELOG.md`, where you replace the `Unreleased` with the appropriate version you are publishing.
 
-_Note: Ensure you set up a [GitHub Token](https://github.com/skywinder/github-changelog-generator/#github-token) when using the changelog generator, or else it will not work properly._
+_Note: Ensure you set up a GitHub Token when using the changelog generator, or else it will not work properly._
```

Comment:
I haven't found this to be an issue (generally speaking)

Reply:
@rwjblue You probably already have a token in GITHUB_AUTH environment variable, though?